libs/SmartStore/SmartStore/Classes/SFQuerySpec.m

```diff
@@ -296,10 +318,19 @@ - (NSString*)computeWhereClause {
 
         case kSFSoupQueryTypeMatch:
             return [@[@"WHERE ",
-                      [self computeSoupFtsReference], @".", DOCID_COL, @" = ", [self computeFieldReference:SOUP_ENTRY_ID], // join clause
-                      @" AND ",
-                      field, @" MATCH '", self.matchKey, @"' "  // match clause -- statement arg binding doesn't seem to work so inlining matchKey
-                      ] componentsJoinedByString:@""];
+                      [self computeFieldReference:SOUP_ENTRY_ID],
```

Comment:
See description of https://github.com/forcedotcom/SalesforceMobileSDK-iOS/pull/1568/commits/95486afffcdc236de4161aa473dad70bc9df2180 for more information